src/utils/ItemLocaleUtils.ts

```diff
@@ -0,0 +1,20 @@
+import {localizedLanguagesStrings} from '../i18n';
```

Comment:
Name the file better. LocaleUtils should be good enough

Reply:
Renamed